README.md

```diff
@@ -4,15 +4,14 @@ Whole-body PBPK model of fluvoxamine as an inhibitor of CYP2C19 and CYP1A2
 This repository contains:
 
 - a PK-Sim snapshot (*.json) file of the current PBPK model
+- the PK-Sim project of the current PBPK model created with OSP version 10
+- Evaluation report created with OSP version 10
```

Comment:
Why adding evaluation report **here**?
It should be added either to the PBPBK-Models_Library-repo or to the Next Release of the Fluvoxamine Model in this Repo(1.2?).